demos/containers/rapids-demo.md

```diff
@@ -198,20 +219,29 @@ All `ubuntu16.04` images use `gcc 5.4`
 **CUDA 9.2**
 
 | Short Tags | Full Tag | Image Type | Python Version |
-| --- | --- | --- |
-| `cuda9.2-base-ubuntu16.04` | `0.9-cuda9.2-base-ubuntu16.04` | base | 3.6 |
-| `latest`<br>or<br>`cuda9.2-runtime-ubuntu16.04` | `0.9-cuda9.2-runtime-ubuntu16.04` | runtime | 3.6 |
-| --- | `0.9-cuda9.2-devel-ubuntu16.04-py3.6` | devel | 3.6 |
-| --- | `0.9-cuda9.2-devel-ubuntu16.04-py3.7` | devel | 3.7 |
+| --- | --- | --- | --- |
+| `cuda9.2-base-ubuntu16.04` | `0.10-cuda9.2-base-ubuntu16.04` | base | 3.6 |
+| `latest`<br>or<br>`cuda9.2-runtime-ubuntu16.04` | `0.10-cuda9.2-runtime-ubuntu16.04` | runtime | 3.6 |
+| --- | `0.10-cuda9.2-devel-ubuntu16.04-py3.6` | devel | 3.6 |
```

Comment:
These are right but they are now in different repos. Not sure if we need to put that here or up above?